resources/quality/nwa3d_a31_e.inst.cfg

```diff
@@ -0,0 +1,125 @@
+[general]
+version = 4
+name = 0.6 Engineering Quality
```

Comment:
It's very uncommon to see the nozzle size in the name of the global quality. Is this correct or is it a mistake?

Reply:
Yes, it's correct. The Engineering Quality is best for the 0.6 Nozzle Variant, so it will help people to know the correct default profile for that variant.

I moved the quality profiles into their own folder. Thanks!